src/EditorFeatures/Core/CommandHandlers/IntelliSenseCommandHandler.cs

```diff
@@ -12,6 +13,7 @@ namespace Microsoft.CodeAnalysis.Editor.CommandHandlers
     [Export(typeof(VSCommanding.ICommandHandler))]
     [ContentType(ContentTypeNames.RoslynContentType)]
     [Name(PredefinedCommandHandlerNames.IntelliSense)]
+    [Order(After = PredefinedCompletionNames.CompletionCommandHandler)]
     internal sealed class IntelliSenseCommandHandler : AbstractIntelliSenseCommandHandler
```

Comment:
i'm a little confused aobut how to intuit what's going on here.   We have our IntelliSense command handler... but it comes after the Completion command handler?   Can you provide clarity about what this means?  What would happen if we came first?  why are there two completion handlers?  etc. etc.

Reply:
it seems no need to support this. will try to remove

---
In reply to: [222070790](https://github.com/dotnet/roslyn/pull/29016#discussion_r222070790) [](ancestors = 222070790)